### ISSUE TITLE
text/plain part: Ways to further improve HTML display of plain text mails

### ISSUE THREAD

**Issue by Djfe** [thundernest/k-9#3171]
This is related to:
#3168 and #3169 
Some ideas:

E-Mail addresses should also be converted to links
```
<test@test.com>
```
but not links for the web browser, rather -> new mail in k9 or add to contacts

example (it's often used in front of quoted replies):
```
Am 08.02.18, 11:25 schrieb "someone" <abc@xyz.de>:
```

**Comment by Djfe** [comment]
dividers, I know of, that the current implementation doesn't convert:
Mails from xda's forum use * and ~ for different kinds of dividers

kind of like in markdown, maybe
text with one * or _ could be italics and two ** or __ could be bold
```
*this could be italic text*
**this could be bold text**
```
I'm not sure whether ~ should ~~strike~~ words

![screenshot_20180208-125050](https://user-images.githubusercontent.com/5685512/35971587-decd9364-0cce-11e8-92cb-7baea8ccc903.png)

**Comment by Djfe** [comment]
kind of like in markdown text with one * or _ could be italics and two ** or __ could be bold
```
*this could be italic text*
**this could be bold text**
```
I'm not sure whether ~ should ~~strike~~ words though

**Comment by Djfe** [comment]
kind of like in markdown text with one * or _ could be italics and two ** or __ could be bold
```
*this could be italic text*
**this could be bold text**
```
I'm not sure whether ~ should ~~strike~~ words though

**Comment by Djfe** [comment]
```
* point 1
* point 2
* point 3
```
should be converted to unordered lists:
* point 1
* point 2
* point 3

**Comment by Djfe** [comment]
What about headings like this one? (forwarded messages)
-------- Ursprüngliche Nachricht --------
Von: "someone" <some@one.de>
Gesendet: 11. Februar 2019 16:49:58 MEZ
An: Me <me@one.de>, someone else <else@one.de>
Betreff: [maillinglist] something

**Comment by Djfe** [comment]
In general texts like these could be converted to headings
```
<h2>
```
```
-- DEUTSCH
-------- Ursprüngliche Nachricht --------
```

**Comment by Djfe** [comment]
I created a new issue for the remainder of this feature request: 
#6242
The rest is addressed in #3608 already.

Monospace rendering of plain text mails is not so important to me anymore. (I rarely use trac)
